src/state/routing/slice.ts

```diff
@@ -13,11 +13,12 @@ export enum RouterPreference {
   AUTO = 'auto',
   API = 'api',
   CLIENT = 'client',
-
-  // Used internally for token -> USDC trades to get a USD value.
-  PRICE = 'price',
 }
 
+// This is excluded from `RouterPreference` enum because it's only used
+// internally for token -> USDC trades to get a USD value.
+export const INTERNAL_ROUTER_PREFERENCE_PRICE = 'price' as const
```

Comment:
nit: something like USD_PRICE might be easier, took me scrolling till here to understand the var change in the files above

Reply:
FWIW we will be moving that completely in a follow-up PR. I agree it is not the best name, but it is being used as a substitute for `RouterPreference` enum, so I wanted to make it as similar in name as possible. That way `RouterPreference | typeof INTERNAL_ROUTER_PREFERENCE_PRICE` reads relatively okay while scanning the codebase.